lib/perl/Genome/Model/Tools/FastTier/TierBitmasks.pm

```diff
@@ -116,174 +61,67 @@ sub create {
+
+        #next build up the regulatory annotated regions
+        my $regulatory_regions = $self->create_regulatory_bitmask(\%genome);
+        print STDERR "Calculated Tier2 regulatory regions\n";
+        $self->debug_message("Calculated Tier2 regulatory regions\n");
+
+
+        $self->in_place_difference_genomes($regulatory_regions, $repeatmasker_regions); 
+        print STDERR "Calculated Tier2 regulatory regions / repeatmasker\n";
+        $self->debug_message("Calculated Tier2 regulatory regions / repeatmasker\n");
+
+        $self->in_place_union_genomes($tier2, $regulatory_regions);
+        print STDERR "Calculated Tier2 conserved U regulatory regions / repeatmasker\n";
+        $self->debug_message("Calculated Tier2 conserved U regulatory regions / repeatmasker\n");
+        $self->in_place_difference_genomes($tier2, $tier1); #exclude things hitting Tier1
+        print STDERR "Calculated (Tier2 conserved U regulatory regions / repeatmasker) / Tier1\n";
+        $self->debug_message("Calculated (Tier2 conserved U regulatory regions / repeatmasker) / Tier1\n");
+        $self->in_place_difference_genomes($tier2, \%genome); #account for masking
+        print STDERR "Calculated (Tier2 conserved U regulatory regions / repeatmasker) / Tier1 / masked genome\n";
+        $self->debug_message("Calculated (Tier2 conserved U regulatory regions / repeatmasker) / Tier1 / masked genome\n");
     }
-
-    $self->in_place_difference_genomes($regulatory_regions, $repeatmasker_regions); 
-    #in_place_difference_genomes($regulatory_regions, $tier2);
-    print STDERR "Calculated Tier2 regulatory regions / repeatmasker\n";
-    $self->debug_message("Calculated Tier2 regulatory regions / repeatmasker\n");
-    #printf "Tier2 regulatory set encompasses %u bases. %f%% of the genome\n", bases_covered($regulatory_regions), bases_covered($regulatory_regions)/$masked_genome_size * 100;
-
-    $self->in_place_union_genomes($tier2, $regulatory_regions);
-    print STDERR "Calculated Tier2 conserved U regulatory regions / repeatmasker\n";
-    $self->debug_message("Calculated Tier2 conserved U regulatory regions / repeatmasker\n");
-    $self->in_place_difference_genomes($tier2, $tier1); #exclude things hitting Tier1
-    print STDERR "Calculated (Tier2 conserved U regulatory regions / repeatmasker) / Tier1\n";
-    $self->debug_message("Calculated (Tier2 conserved U regulatory regions / repeatmasker) / Tier1\n");
-    $self->in_place_difference_genomes($tier2, \%genome); #account for masking
-    print STDERR "Calculated (Tier2 conserved U regulatory regions / repeatmasker) / Tier1 / masked genome\n";
-    $self->debug_message("Calculated (Tier2 conserved U regulatory regions / repeatmasker) / Tier1 / masked genome\n");
-    printf "Tier2 encompasses %u bases. %f%% of the genome\n", $self->bases_covered($tier2), $self->bases_covered($tier2)/$masked_genome_size * 100;
+    printf $self->report_coverage("Tier2", $tier2, $masked_genome_size);
```

Comment:
This should now be a `print` instead of `printf` since the formatting is handled inside `report_coverage`.